RecoTauTag/RecoTau/plugins/PFRecoTauDiscriminationAgainstMuon2.cc

```diff
@@ -200,9 +200,10 @@ double PFRecoTauDiscriminationAgainstMuon2::discriminate(const reco::PFTauRef& p
 	const reco::PFCandidate* pflch = dynamic_cast<const reco::PFCandidate*>(pfLeadChargedHadron.get());
 	if (pflch != nullptr) {
 	  reco::MuonRef muonRef = pflch->muonRef();      
-	  if (muonRef.isNonnull() && muon == pflch->muonRef() )
-	  if ( verbosity_ ) { edm::LogPrint("PFTauAgainstMuon2") << " matches muonRef of tau --> skipping it."; }
-	  continue;
+	  if (muonRef.isNonnull() && muon == pflch->muonRef() ) {
+	    if ( verbosity_ ) { edm::LogPrint("PFTauAgainstMuon2") << " matches muonRef of tau --> skipping it."; }
```

Comment:
Hi, 

do I read it correctly that the issue was just die to missing braces? Did oyu add the edm::LogPrint then in a commit before? It's good that the issue was caused (and fixed) by so trivial changes. 

Cheers,
Roger

Reply:
Yes, it was my mistake when implementing it. I think I got confused by the braces around the LogPrint, which were there before (and I think it's good practice to always add them so that was fine) plus the usual funny emacs default indentation in much of the code